README.md

```diff
@@ -37,6 +37,38 @@ and treat as best.
 A Polkadot [collator](https://wiki.polkadot.network/docs/en/learn-collator) for the parachain is
 implemented by the `polkadot-parachain` binary (previously called `polkadot-collator`).
 
+### Relaychain Interaction
+To operate a parachain, we need a connection to the corresponding relaychain.
+Currently, there are two options available:
+1. Use an in-process relaychain full-node
+2. Connect to an external relaychain node via websocket RPC
+
+#### In-process Relaychain Node
+This is the default. If a relaychain chain spec is passed to the node via command line arguments, the collator node spawns an in-process full node. This full node has all the typical components, including all polkadot subsystems, its own substrate networking stack, and the relaychain runtime.
```

Comment:
Hmm... 

> If a relaychain chain spec is passed to the node via command line arguments, the collator node spawns an in-process full node.

This doesn't seem to be strictly true as you're also passing the relaychain spec in the example for the external node too, and over there it obviously doesn't spawn a full node in-process (instead it spawns a cut down one). (:

Maybe something like this?

```suggestion
By default if an external relaychain node is not specified then a full relaychain node will be spawned within the same process.

This node has all of the typical components of a normal Polkadot node, and will have to fully sync with the relaychain to work.
```